src/main/webapp/scripts/script.js

```diff
@@ -134,3 +129,147 @@ function geocodeLatLng(position) {
     }
   );
 }
+
+var body_colors=new Array("#E8E8E8", "#CFCFCF", "#E6E4C1", "#DEBDBD", "#B4D1B4",
```

Comment:
As far as I can see the very same code lives in http://usefulscript.ru/gradient_fon.php. While reusing code that you have not written can be OK, similarly as existing reusing libraries can be OK, you need to answer the question about 1) copyright and licensing. Also 2) you should tell your reviewers where the code is coming from or otherwise we may believe that you have written the code yourself. We can talk a bit offline.